src/library/scala/math/package.scala

```diff
@@ -63,6 +63,9 @@ package scala
   *
   * @groupname randomisation Pseudo Random Number Generation
   * @groupprio randomisation 130
+  *
+  * @groupname exact Functions to avoid overflow on primitive types
```

Comment:
@SethTisue - Change to

    @groupname exact Exact Arithmetic
    @groupdesc exact Integral addition, multiplication, stepping and conversion throwing ArithmeticException instead of underflowing or overflowing